### PR TITLE
ZENKO-125 ft: mongo prometheus integration

### PR DESCRIPTION
**Pull request by ssalaues**
Review only for now.

This bumps the MongoDB chart to the latest and has prometheus metrics capabilities. However with this comes a bump to MongoDB 3.6 and needs to be taken into consideration before merging.

Important NOTE:
**So far there have been issues with upgrades. It may be possible to fix this with custom images for the init containers but requires validation.**

### CODE REVIEW

**Comment by ssalaues** [comment]
@bert-e wait

**Comment by ssalaues** [comment]
The mongodb chart has been switched to our fork of the chart that allows for 3.4 compatibility and capable helm upgrades to the upstream chart with mongodb version 3.6.

It's using the scality/charts as a repo currently but would prefer to use scality/Zenko as the helm repository. So far I haven't been able to get GitHub pages to work using the gh-pages branch.

File: charts/README.md [hunk 1]
```diff
@@ -3,15 +3,16 @@ Zenko Helm Chart
...
+assumes you have cloned the Zenko repository and already `cd` into this directory.
 
 First, retrieve all dependencies:
 
 ```console
 $ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
 "incubator" has been added to your repositories
 
-$ helm repo add scality https://scality.github.io/charts/
+$ helm repo add scality https://scality.github.io/Zenko/
```
Comment (ssalaues) [inline charts/README.md:15]: @NicolasT Is preferred than to use the `scality/charts` fork?

**Review by giacope — APPROVED** [review]

File: charts/minikube.md [hunk 2]
```diff
@@ -43,6 +43,9 @@ Once you have the repo cloned you can retrieve all dependencies:
 $ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
 "incubator" has been added to your repositories
 
+$  helm repo add scality https://scality.github.io/Zenko/
+"scality has been added to your repositories
```
Comment (jonathan-gramain) [inline charts/minikube.md]: missing quote in `"scality`

**Review by jonathan-gramain — APPROVED** [review]

File: charts/minikube.md (hunk 2 above)
Comment (ssalaues) [reply charts/minikube.md]: Fixed! Thank you

**Comment by ssalaues** [comment]
Tried testing with an off the shell mongodb grafana dashboard that is from one of the mongodb exporter repos but couldn't get it to work with our metrics likely will need a custom one. We'll explore it further in ZENKO-511.

**Pull request merged**